recipes/opsin/1.4.0/opsin.sh

```diff
@@ -42,7 +42,7 @@ for arg in "$@"; do
     esac
 done
 
-if [ "$jvm_mem_opts" == "" ]; then
+if [ "$jvm_mem_opts" == "" ] && [ -z ${_JAVA_OPTIONS+x} ]; then
```

Comment:
@chapmanb what do you think about this? This would enable us to overwrite the settings globally without touching the commandline.